app/src/testDebug/kotlin/org/cru/godtools/ui/languages/LanguageSettingsLayoutPaparazziTest.kt

```diff
@@ -22,7 +22,7 @@ class LanguageSettingsLayoutPaparazziTest(
         appLanguage = Locale.FRENCH,
         appLanguages = 1234,
         downloadedLanguages = persistentListOf(
-            Language(Locale.ENGLISH),
+            Language(Locale.ENGLISH, "English - United States", isForcedName = true),
```

Comment:
add this as an additional language and don't replace one of the existing ones

Reply:
I actually think i would prefer there to be a separate test class that tests the `LanguageName` composable directly